### ISSUE TITLE
Unable to fresh install Thor 1.46.14

### ISSUE THREAD

**Issue by lscheffler** [VFPX/Thor#205]
## 📝 Provide detailed reproduction steps (if any)

1. Download Thor via
  - https://github.com/VFPX/Thor/archive/master.zip or 
  - https://github.com/VFPX/Thor/blob/master/ThorUpdater/Thor.zip
3. Expand into an empty folder
4. Start VFP without Thor
5. Run Thor.App from the folder created in [3]

### ✔️ Expected result

Should update

### ❌ Actual result

Fails with error message

## 📷 Debugging Info
Error message:
```
Locate class
Error instantiating class.  Cannot find e:\se\tmp\thor\thor\tools\procs\thor_sfsplitter.vcx.
Locate...
Ignore
Cancel
Help
```
Debugger stuff:
![grafik](https://github.com/VFPX/Thor/assets/27769909/aa4e86ff-9118-42e8-8511-9fd4501830f9)

## Hint:
`thor_sfsplitter.vcx` is somehow not in *thor\thor\tools\procs* but in *thor\thor\tools*   
The file was moved from *Installed Files/Source/ToolsToInstall/thor_sfsplitter.vct* to * Installed Files/Thor/Tools/thor_sfsplitter.vct* on 2023-06-28 11:41:51 in commit 95b483fe0a7.

It is also to note that there is a *thor\thor\tools\thor_sfsplitter.vcx* and a *thor\source\thor_sf_splitter.vcx*, the libs look very similar.

## Update
I found an install from *Thor - 1.46.02 - August 19, 2023*, this installs to itself. A subsequent CFU runs.

**Comment by lscheffler** [comment]
## Update 2
O.k. some more information: 
If one do:
1. Install Thor Thor - 1.46.02  as above.
2. Install Updates as suggested
3. Run CFU   

all is fine.   

---
If one do:
1. Install Thor Thor - 1.46.02  as above.
2. Do not Install Updates 
3. Run CFU   

it fails as well.   

---
If you like to test with 1.46.02, follow <a href="https://github.com/VFPX/Thor/tree/79fd65aebe106c150e5bf2b5f5bda4e9ad9c6d68" target="_blank">this link</a>, and download the repo as a zip (In the green Code thing).   

I have no idea with which version this will change, I just run the test.

**Comment by Jimrnelson** [comment]
This is fixed with version 1.46.15 (today)
